Tasks/3 - Condition/condition.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head></head>
+  <body>
+    <script>
+  var answer = prompt("What do you want to eat?");
+  if (answer === "seed") {
+  alert("Nice choice!");
+} else if (answer === "bread") {
```

Comment:
Please, align code. Copy and paste code [here](https://codebeautify.org/jsviewer), for example

Reply:
Ok, you aligned JS code itself, but now you should align to the right all JS code up to `<script>` tag

Like this for example:
![image](https://user-images.githubusercontent.com/30930391/216860726-da6b5098-20ae-4140-9c55-05a6b3f0712f.png)